src/applications/personalization/profile/tests/components/direct-deposit/BankInfo.unit.spec.jsx

```diff
@@ -110,9 +110,18 @@ describe('DirectDepositCNP', () => {
     server.close();
   });
 
+  let bankTypeBeingEdited = null;
+  const setBankTypeBeingEdited = bankType => {
+    bankTypeBeingEdited = bankType;
+  };
+
   const ui = (
     <MemoryRouter>
-      <BankInfo type={benefitTypes.CNP} />
+      <BankInfo
+        type={benefitTypes.CNP}
+        setBankTypeBeingEdited={setBankTypeBeingEdited}
+        bankTypeBeingEdited={bankTypeBeingEdited}
```

Comment:
...so you can just pass in `setBankTypeBeingEdited={() => {}}` and `bankTypeBeingEdited={null}`

Reply:
I will test that out. I thought I did that already and it had problems, but I will check again. Also will look into the UI error

Reply:
In playing with the Direct Deposit section locally in Cypress, even on `master`, not this branch, the browser alert never shows up when changing Profile sections while the bank info form is "dirty." Basically `master` in Cypress does not behave the way it does on staging. On staging when you start editing bank info, enter a routing number, and then click to another section of the Profile, you see a browser prompt asking you if you are sure you want to leave. (this prompt comes from https://github.com/department-of-veterans-affairs/vets-website/blob/bc04b4c54aa88cb8a22223b47362a41b914d4573/src/applications/personalization/profile/components/direct-deposit/BankInfo.jsx#L412-L415). _But this prompt never appears in Cypress_. I think we should really look into why that might be, because making sure that prompt appears at the right time is pretty important.

Reply:
The prompt is another issue, but I haven't noticed it not working. It seems to work for me, but there is a ticket to make sure contact information and direct deposit has that fix. I also really hate that the prompt is not custom, but I did find an article to use a custom prompt, but also another ticket I think? Anyways, I can't reproduce your video. Here's mine. Did I miss a step?

https://user-images.githubusercontent.com/84803567/139738080-c49ed600-4ec4-477b-bfcf-8411d9ef22f5.mp4

.